code/game/objects/items/nemesis_items.dm

```diff
@@ -0,0 +1,458 @@
+	icon_state = "trap_explosion"
+	duration = 8
+
+/obj/effect/temp_visual/nemesis_circle/huge
+	name = "big energy blast"
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "nemesis_circle"
+	duration = 5
+	pixel_x = -32
+	pixel_x = -32
+
+/obj/item/shield/energy/nemesis
+	name = "tactical energy shield"
+	desc = "A tactical wrist-mounted energy shield from Nemesis Solutions."
+	icon = 'icons/obj/nemesis.dmi'
+	icon_state = "nemesis0"
+	base_icon_state = "nemesis"
+	force = 3
+
+	item_flags = SLOWS_WHILE_IN_HAND
+	slowdown = 0
+
+/obj/item/shield/energy/nemesis/Initialize()
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+/obj/item/shield/energy/nemesis/attack_self(mob/living/carbon/human/user)
+	if(clumsy_check && HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
+		to_chat(user, "<span class='userdanger'>You beat yourself in the head with [src]!</span>")
+		user.take_bodypart_damage(5)
+
+	var/obj/item/clothing/gloves/rapid/nemesis/gloves = user.get_item_by_slot(ITEM_SLOT_GLOVES)
+	if(!gloves || !istype(gloves))
+		return FALSE
+
+	if(!gloves.charge)
+		to_chat(user, "<span class='warning'>[src] fails to activate without any charge avalible.</span>")
+		return FALSE
+
+	active = !active
+	icon_state = "[base_icon_state][active]"
+
+	if(active)
+		force = on_force
+		throwforce = on_throwforce
+		throw_speed = on_throw_speed
+		w_class = WEIGHT_CLASS_BULKY
+		playsound(user, 'sound/weapons/saberon.ogg', 35, TRUE)
+		to_chat(user, "<span class='notice'>[src] is now active.</span>")
+		slowdown = 2 //Heavy slowdown
+	else
+		force = initial(force)
+		throwforce = initial(throwforce)
+		throw_speed = initial(throw_speed)
+		w_class = WEIGHT_CLASS_TINY
+		playsound(user, 'sound/weapons/saberoff.ogg', 35, TRUE)
+		to_chat(user, "<span class='notice'>[src] can now be concealed.</span>")
+		slowdown = 0
+	add_fingerprint(user)
+
+/obj/item/shield/energy/nemesis/on_shield_block(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", damage = 0, attack_type = MELEE_ATTACK)
+	var/obj/item/clothing/gloves/rapid/nemesis/gloves = owner.get_item_by_slot(ITEM_SLOT_GLOVES)
+	if(!gloves || !istype(gloves) || gloves.charge <= round(damage / 10))
+		active = !active
+		icon_state = "[base_icon_state][active]"
+		force = initial(force)
+		w_class = WEIGHT_CLASS_TINY
+		playsound(owner, 'sound/weapons/saberoff.ogg', 35, TRUE)
+		to_chat(owner, "<span class='warning'>[src] turns off!</span>")
+		slowdown = 0
+
+	if(istype(gloves))
+		if(damage < 10)
+			gloves.lose_charge(1)
+			return
+		gloves.lose_charge(round(damage * 0.1))
+
+/obj/item/shield/energy/nemesis/IsReflect()
+	if(!ishuman(loc))
+		return (active)
+
+	var/mob/living/carbon/human/owner = loc
+	var/obj/item/clothing/gloves/rapid/nemesis/gloves = owner.get_item_by_slot(ITEM_SLOT_GLOVES)
+	if(!gloves || !istype(gloves))
+		return (active)
+
+	if(gloves.charge < 2)
+		if(active)
+			active = !active
+			icon_state = "[base_icon_state][active]"
+			force = initial(force)
```

Comment:
you forgot to reset throwforce and throw_speed

Reply:
It's NODROP, no need to